apps/nuxt-ssr/pages/_schema/ssr-catalogue/cohorts/index.vue

```diff
@@ -28,24 +29,30 @@
 </template>
 
 <script>
-import cohortsQuery from "../../../../store/gql/cohorts.gql";
```

Comment:
I have discussed with Joris and Max to go back to double quotes to not need this random format changes. Change not merge unfortunately.

Reply:
please update lint settings